lib/api/repository/survey_repository.dart

```diff
@@ -34,7 +33,9 @@ class SurveyRepositoryImpl extends SurveyRepository {
   }
 
   void _saveSurveysCache(int pageNumber, List<SurveyModel> surveyModels) {
```

Comment:
What do you think about changing `int pageNumber` to `bool wantsClearCache`, passing a `pageNumber` here is quite redundant 🤔